js/src/hooks/useRaiseBudgetRecommendations.js

```diff
@@ -24,11 +24,13 @@ import useGoogleAdsAccount from './useGoogleAdsAccount';
 /**
  * Retrieves campaigns with budget recommendations to raise.
  *
+ * @param {number|null} campaign_id Optional. The ID of the campaign to filter recommendations. Default is null.
+ *
  * @return {Object} An object containing:
  *   - {Array<CampaignRecommendation>} campaigns: An array of campaigns with budget recommendations, or an empty array if none.
  *   - {boolean} hasFinishedResolution: Whether the recommendations resolution has completed.
  */
-const useRaiseBudgetRecommendations = () => {
+const useRaiseBudgetRecommendations = ( campaign_id = null ) => {
```

Comment:
```suggestion
const useRaiseBudgetRecommendations = ( campaign_id ) => {
```
@ankitrox I think we can omit the default `null` value here since `campaign_id` will be `undefined`.

Reply:
If we omit this, it will give error as it will be expecting `campaign_id` in `js/src/pages/dashboard/all-programs-table-card/index.js` [here](https://github.com/woocommerce/google-listings-and-ads/blob/e71415e90a19550a31bf2b6afb24a3ed0ea7b9b8/js/src/pages/dashboard/all-programs-table-card/index.js#L86), which should not be mandatory.

Reply:
Update the signature to the one as suggested.